src/demo-app/demo-app-module.ts

```diff
@@ -20,7 +20,7 @@ import {AccessibilityDemoModule} from './a11y/a11y-module';
 @NgModule({
   imports: [
     BrowserModule,
-    BrowserAnimationsModule,
+    NoopAnimationsModule,
```

Comment:
We don't want to switch the demo-app completely to noop. It should only be for some specific section you want to test.

Reply:
Whoops.